### PR TITLE
add: documentation for signer name + grammar nits

### PR DESCRIPTION
**Pull request by zachterrell57**
Added documentation for `name` field on Signer, and also a few grammar nits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,28 +11,58 @@ Farcaster is a protocol for building decentralized social applications. If you w
...
-   1. [Synchronization](#41-synchronization)
-5. [Applications](#5-applications)
-6. [Upgradeability](#6-upgradeability)
-   1. [Release Schedule](#61-release-schedule)
-7. [Acknowledgements](#7-acknowledgements)
-8. [Appendix A: Proposals](#8-appendix-a-proposals)
-9. [Appendix B: Delta Specifications](#9-appendix-b-delta-specifications)
-10. [Appendix C: Application Specifications](#10-appendix-c-application-specifications)
-11. [Appendix D: Contract Specifications](#11-appendix-d-contract-specifications)
+- [Farcaster Protocol](#farcaster-protocol)
```
Comment (zachterrell57) [inline README.md:14]: No idea why my linter expanded the TOC like this.....lmk if you want me to change it

Comment (varunsrin) [reply README.md:14]: yes please, i intentionally kept the toc small

**Review by varunsrin — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -8,8 +8,9 @@ Farcaster is a protocol for building decentralized social applications. If you w
 2. a quick video overview, check out the [video series](https://www.youtube.com/watch?v=vMWjol6xHJ0&list=PL0eq1PLf6eUdm35v_840EGLXkVJDhxhcF)
 3. to contribute to the protocol, see the [contributing guidelines](CONTRIBUTING.md)
 4. the latest updates, join the [farcaster telegram channel](https://t.me/farcasterxyz)
+<!--
 
-## Table of Contents
+## Table of Contents -->
```
Comment (varunsrin) [inline README.md:13]: toc should not be commented out

**Review by varunsrin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (zachterrell57) [reply README.md:13]: whoops

**Pull request merged**